doc/synch_web.py

```diff
@@ -51,12 +52,12 @@ def body_updater(match):
 
     mkdir(destination + root)
     dest_path = os.path.join(destination + root, os.path.basename(path))
-    with open(path, 'rb') as fpi:
```

Comment:
Are you sure that we don't want these in binary mode?

Reply:
Yes, all source text files in that docs folder are now utf-8, otherwise it's a PITA to work in binary mode in Python3

Reply:
Does it work on Windows with the existing line endings?

Reply:
Well it writes out CRLF line endings, but that's generally fine depending on git settings. I've changed it anyway to just always write out LF line endings

Reply:
Ah, okay.